src/cpp/session/modules/SessionConnections.R

```diff
@@ -300,7 +311,7 @@ options(connectionObserver = list(
                paste(tolower(driver), ".R", sep = "")
             )
             
-            if (file.exists(snippetsFile)) {
+            if (identical(file.exists(snippetsFile), TRUE)) {
```

Comment:
That's a good point, it actually is defined as `identical(x, TRUE)`. Would you mind taking this PR as is? I would prefer to avoid retesting this one, but will consider this on future PRs!